src/gtpu/encap.c

```diff
@@ -736,11 +739,17 @@ int update_urr_counter_and_send_report(struct pdr *pdr, struct far *far, u64 vol
 
         for (i = 0; i < report_num; i++) {
             // TODO: FAR ID for Quota Action IE for indicating the action while no quota is granted
-            if (triggers[i] == USAR_TRIGGER_START){
+            if (triggers[i] & URR_RPT_TRIGGER_START){
```

Comment:
It should be rollback because
https://github.com/free5gc/gtp5g/pull/143/files#diff-592a60b4467f2f04f2aa1e5919b9cc9ff7179423a7dcfe2e54ac9687f2d0722bR691